helm/hpcc/templates/thor.yaml

```diff
@@ -348,8 +348,8 @@ spec:
         accessEsp: {{ $commonCtx.eclAgentUseChildProcesses | ternary "yes" "no" | quote }}
         app: "thor"
         component: "thor-eclagent"
-        helmVersion: 8.8.0-closedown0
         instance: {{ $commonCtx.eclAgentName | quote }}
+{{ include "hpcc.addStandardLabels" (dict "root" $ "component" "thor-eclagent" "name" "thor" "instance" $commonCtx.eclAgentName ) | indent 8 }}
```

Comment:
component="eclagent"

Reply:
ok but this seems to contradict an earlier change you suggested where the app/component was roxie/server changed to roxie/roxie-server

in this case thor/thor-eclagent changed to thor/eclagent...